webapp/src/components/modals/channel_subscriptions/edit_channel_subscription.tsx

```diff
@@ -1,9 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+// TODO: i18n
```

Comment:
This file and `webapp/src/components/modals/channel_subscriptions/select_channel_subscription.tsx` still need to be fully translated